airbyte-webapp/src/components/EntityTable/hooks.tsx

```diff
@@ -27,13 +29,17 @@ const useSyncActions = (): {
 
     const frequency = getFrequencyConfig(connection.schedule);
 
-    analyticsService.track("Source - Action", {
-      action: connection.status === "active" ? "Disable connection" : "Reenable connection",
+    const trackableAction =
+      connection.status === ConnectionStatus.active ? TrackActionActions.DISABLE : TrackActionActions.REENABLE;
+
+    const trackableActionString = `${trackableAction} connection`;
+
+    trackSourceAction(trackableActionString, [trackableAction], {
+      frequency: frequency?.type,
       connector_source: connection.source?.sourceName,
-      connector_source_id: connection.source?.sourceDefinitionId,
-      connector_destination: connection.destination?.destinationName,
+      connector_source_definition_id: connection.source?.sourceDefinitionId, //another place I'm changing a label... clarify if that's ok
```

Comment:
@nataliekwong there are some places where the `source definition id` was being sent with the label `connector_source_id` and other places it was being sent with the label `connector_source_definition_id`.

Is it ok to change them to be uniform (`connector_source_definition_id`)?  Again, don't want to break any existing analytics here.

Reply:
Yeah, let's align these to be consistent (no worries about breaking things - we can refactor the analytics afterwards). Thanks for catching this!